### PR TITLE
UICIRC-867: UI tests replacement with RTL/Jest for CheckoutSettingsForm

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
UI tests replacement with RTL/Jest for CheckoutSettingsForm

## Refs
https://issues.folio.org/browse/UICIRC-867

### CODE REVIEW

File: test/jest/__mock__/stripesCore.mock.js [hunk 1]
```diff
@@ -12,4 +12,5 @@ jest.mock('@folio/stripes/core', () => ({
   stripesShape: {},
   withStripes: (Component) => (props) => <Component {...props} />,
   IfPermission: jest.fn(({ children }) => <div>{children}</div>),
+  useCustomFields:jest.fn(() => []),
```
Comment (artem-blazhko) [inline test/jest/__mock__/stripesCore.mock.js:15]:
```suggestion
  useCustomFields: jest.fn(() => []),
```

File: src/settings/CheckoutSettings/CheckoutSettingsForm.test.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+import '../../../test/jest/__mock__';
+
+import CheckoutSettingsForm, {
+  getCustomFieldPatronIdentifiers,
+} from './CheckoutSettingsForm';
+
+jest.mock('@folio/stripes/core', () => ({
+  useCustomFields: jest.fn(() => []),
+}));
```
Comment (artem-blazhko) [inline src/settings/CheckoutSettings/CheckoutSettingsForm.test.js:15]: Looks like it is not necessary here. You already added this mock to `test/jest/__mock__/stripesCore.mock.js` file.

File: src/settings/CheckoutSettings/CheckoutSettingsForm.test.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+  };
+  const labelIds = {
+    patronIds: 'ui-circulation.settings.checkout.patronIds',
+    barcode: 'ui-circulation.settings.checkout.identifiers.barcode',
+    externalSystemId: 'ui-circulation.settings.checkout.identifiers.externalSystemId',
+    identifiersId: 'ui-circulation.settings.checkout.identifiers.id',
+    username: 'ui-circulation.settings.checkout.identifiers.username',
+    userCustomFields: 'ui-circulation.settings.checkout.userCustomFields',
+    timeout: 'ui-circulation.settings.checkout.checkin.timeout',
+    timeoutDuration: 'ui-circulation.settings.checkout.timeout.duration',
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/settings/CheckoutSettings/CheckoutSettingsForm.test.js:39]: removed

File: test/jest/__mock__/stripesCore.mock.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply test/jest/__mock__/stripesCore.mock.js:15]: Fixed

File: src/settings/CheckoutSettings/CheckoutSettingsForm.test.js (hunk 2 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/CheckoutSettings/CheckoutSettingsForm.test.js:15]: removed

**Review by artem-blazhko — APPROVED** [review]

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**